frontend/packages/topology/src/utils/removeConnection.tsx

```diff
@@ -19,10 +21,10 @@ export const removeConnection = (edge: Edge): Promise<any> => {
       </>
     ),
     message,
-    btnText: 'Delete',
+    btnText: t('topology~Delete'),
     submitDanger: true,
     executeFn: () => {
-      return removeTopologyResourceConnection(edge).catch((err) => {
+      return removeTopologyResourceConnection(t, edge).catch((err) => {
```

Comment:
I'd prefer to have `confirmModal` accept a callback which is either executed in the render context so that the function can use `useTranslation` or it gets passed in the translation function. This would make it easier to use these modal launchers without the need for the consumer to provide it's own `t` function.